test/python/transpiler/test_pass_scheduler.py

```diff
@@ -600,8 +600,83 @@ def test_conditional_and_loop(self):
         self.assertEqual(expected, passmanager.passes())
 
 
+class TestLogPasses(QiskitTestCase):
+    """ Testing the log_passes option """
```

Comment:
For code style and consistency's sake, can you remove all the extra spaces at the beginning and end of docstrings? https://www.python.org/dev/peps/pep-0257/
```suggestion
    """Testing the log_passes option."""
```